PaintingTrivia.py

```diff
@@ -125,35 +144,66 @@ def guess_the_painter():
             return x
 
         try:
-            chosen_painter = int(request.form['chosen_painter'])
+            quiz = pickle.loads(session['quiz'])
+            chosen_painter_id = int(request.form['chosen_painter'])
+            question = quiz.current_question
+            quiz.process_answer(question, chosen_painter_id == question.correct_option.id)
+            session['quiz'] = pickle.dumps(quiz)
         except KeyError:
-            chosen_painter = -1
-        key = 'right_guesses' if chosen_painter == session[
-            'selected_painter_id'] else 'wrong_guesses'
-        session[key] += 1
+            print "Something failed"
 
-    #http://docs.sqlalchemy.org/en/latest/orm/tutorial.html#using-exists
```

Comment:
why did you delete these comments? I mean, i've put them in here on purpose as a reference (for developers only of course). it does not belong to documentation